### ISSUE TITLE
New Entry for Portugal

### ISSUE THREAD

**Issue by eogitbot** [w3c/wai-policies-prototype#425]
New Entry for Portugal

Submitted by [Vera Lange](mailto:vera.lange@han.nl):

~~~yaml
---
country:
  en: Portugal
  # Manual enter other country names: 
updated: 2022-04-20
updatemsg:
# Related page:
province: 
policies:
  - title:
      en: Decreto-Lei n.º 83/2018 de 19 de outubro - Define os requisitos de acessibilidade dos sítios web e das aplicações móveis de organismos públicos, transpondo a Diretiva (UE) 2016/2102
    url: https://dre.pt/home/-/dre/116734769/details/maximized
    updated: 2018
    wcagver: none, 
    enactdate: 2018
    type: law, 
    ministries:
      - title:
          pt: "Ministry of Science, Technology and Higher Education (Portuguese: Ministério da Ciência, Tecnologia e Ensino Superior or MCTES)"
        url:
          pt: "https://www.portugal.gov.pt/pt/gc21/area-de-governo/ciencia-tecnologia-e-ensino-superior"
    webonly: true
    scope: 
    standard:
      - title:
          pt: "EN 301 549"
        desc: "The standard is referenced in the law in article 7."
        url:
          pt: "https://dre.pt/home/-/dre/116734769/details/maximized"
    documents:
      - title:
          en: ""
        desc: ""
        url:
          en: ""
---
~~~

**Comment by brewerj** [comment]
JB reviewed, looks fine

**Comment by iadawn** [comment]
Included in #495
